projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-search/commet-search.component.spec.ts

```diff
@@ -91,7 +91,8 @@ describe('CommentSearch', () => {
 
 @Component({
   selector: `host-component`,
-  template: `<mv-comment-search [annotationSet]="annotationSet"></mv-comment-search>`
+  template: `
+      <mv-comment-search [annotations]="annotationSet"></mv-comment-search>`
 })
 class TestHostComponent {
   annotationSet = { annotations: [] };
```

Comment:
I guess we should pass annotations here now

Reply:
its a spec file